bin/sirius-script-rabpm-mngr.py

```diff
@@ -102,6 +105,8 @@ def main():
     parser = argparse.ArgumentParser(prog="sirius-script-rabpm-mngr",
                                      description="BPM Rack Management Utility")
 
+    parser.add_argument("--verbose", action='store_true')
```

Comment:
I missed the "help" definition.  Something short like "show executed commands output" should be enough and better than nothing, IMHO.